packages/upgrade/src/upgrade.ts

```diff
@@ -42,8 +45,7 @@ export async function upgrade(input: UpgradeInput): Promise<UpgradeOutput> {
     LintPlugin,
   ];
 
-  logger?.info('Upgrade started.');
-  logger?.info(`Base path: ${basePath}`);
```

Comment:
Let's keep the logger at least in this file to show progress, otherwise the user won't have any feedback - if CLI is stuck or working.

Reply:
We should propagate the log stream into commander then